.github/workflows/soaking.yml

```
@@ -20,6 +20,11 @@ on:
   # or we can manually trigger this workflow by using dispatch for debuging
   repository_dispatch:
     types: [bump-version]
+  workflow_dispatch:
+    inputs:
+      sha:
+        description: 'Commit Hash You Want To Soak Must Be The Full SHA'
```

nitpick: For consistency with other workflows. Feel free to ignore.

https://github.com/aws-observability/aws-otel-collector/blob/f15d6496e42745da36bb657f755601cd43b63856/.github/workflows/CD.yml#L14-L24

```suggestion
        description: 'the commit hash to soak test'
```